.github/workflows/linkchecker.yml

```diff
@@ -7,5 +7,12 @@ jobs:
       # Check out the latest version of the code
       - uses: actions/checkout@v4
 
+      # Remove folders that we do not want to check.
+      # This is due to the markdown-link-check action
+      # not having an exclude parameter, see
+      # https://github.com/gaurav-nelson/github-action-markdown-link-check/issues/38
+      - name: Remove folders that we do not want to check
```

Comment:
Isn't this going to just remove an entire folder from the repo? See https://github.com/cosmos/interchain-security/pull/1464 as a potential solution.

Reply:
Yes it is - maybe I'm wrong, but your PR just changes the trigger for the workflow not to trigger on changes to .changelog, but the check still runs on it, right?

Reply:
In this PR, note the folder is only removed for this particular workflow, so it doesn't negatively impact other workflows